docs/site/Dynamic-models-repositories-controllers.md

```diff
@@ -0,0 +1,218 @@
+---
+lang: en
+title: 'How to dynamically add models, repositories, and controllers during runtime'
+keywords: LoopBack 4.0, LoopBack 4, model, repository, controller, tutorial
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Dynamic-models-repositories-controllers-tutorial.html
+---
+
+## How to dynamically add models, repositories, and controllers during runtime
+
+Typically, a LoopBack app's models, repositories, and controllers are defined
+as static files in the project directory. However, there may be scenarios where
+all or some of these might have to be created after the app starts, during
+runtime.
+
+LoopBack provides the ability to dynamically create models, repositories, and
+controllers during runtime. This document is an explanation of how to go about
+doing it.
+
+### Defining a ModelDefinition object
+
+A `ModelDefinition` object is the first step to creating a LoopBack model, it is
+an abstraction for specifying the various attributes of a LoopBack model. A
+`ModelDefinition` object is instantiated by passing a name or a
+[ModelDefinitionSyntax](https://loopback.io/doc/en/lb4/apidocs.repository.modeldefinitionsyntax.html#modeldefinitionsyntax-interface)
```

Comment:
Please use a relative link. I think the following should work, please double check:

```suggestion
[ModelDefinitionSyntax](./apidocs/repository.modeldefinitionsyntax.html#modeldefinitionsyntax-interface)
```

You will most likely need to run `npm run tsdocs` to populate API doc files, otherwise the link will return 404.

The same comment applies to all other places where you are linking to apidocs, please find and update them yourself.